### ISSUE TITLE
[Temporary Fix] For those who receives "sorry we could not configure your system to connect to outline" message

### ISSUE THREAD

**Issue by LBYPatrick** [Jigsaw-Code/outline-client#741]
**FIX**

1. After the "failed" installation, go to control panel and manually rename the TAP device to ``outline-tap0``
2. Run the installer again to force reinstall Outline.
3. Re-enter your server info (or the ss:// link) in the Outline App and connect. 
4. Now everything should work properly.

**Describe the bug**

In some cases (i.e. Windows install language isn't en_us), Outline installer would not configure the name of the virtual TAP device properly,failing the installation process.

**To Reproduce**

See bug description.

**Expected behavior**

The Virtual TAP device would be named as a new "ethernet" device by Windows 10.

**Screenshots**

(N/A)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Microsoft Edge (Chromium)
 - Version 80.0.361.48

**Comment by yuzifen** [comment]
It is working for me

**Comment by Alireza-Honarvar** [comment]
Can see this video for fix error :
https://www.youtube.com/watch?v=K3BcLl_9mvk

**Comment by LBYPatrick** [comment]
My pleasure! Though, is this still an issue after 4 years?!

**Comment by dinosaurtirex** [comment]
> My pleasure! Though, is this still an issue after 4 years?!

Yep, it is. And it's probably because i am installed special windows build which is cut a lot of stuff.
